backoffice/utils.py

```diff
@@ -40,7 +40,9 @@ def group_required(*group_names):
     """Requires user membership in at least one of the groups passed in."""
     def in_groups(user):
         if user.is_authenticated():
-            if bool(user.groups.filter(name__in=group_names)) or user.is_superuser:
+            if (
+                    user.is_superuser or
+                    bool(user.groups.filter(name__in=group_names))):
```

Comment:
Same remark about indentation style, I would prefer:

```python
            if user.is_superuser or \
                   bool(user.groups.filter(name__in=group_names))):
```

Reply:
Same I find your proposal less good for 2 reasons:
- 2 conditions not on par when they should be,
- the parenthese is IMO better than \

Reply:
Yes, sorry for that. I made a new proposal on the previous indentation thread.